src/components/BulkEnrollmentPage/stepper/AddLearnersStep.jsx

```diff
@@ -60,27 +112,30 @@ const AddLearnersStep = ({
         <Link to={`/${enterpriseSlug}/admin/${ROUTE_NAMES.subscriptionManagement}/${subscriptionUUID}`}>{LINK_TEXT}</Link>
       </p>
       <h2>{ADD_LEARNERS_TITLE}</h2>
-      {loading && <TableLoadingSkeleton data-testid="skelly" />}
-      {!loading && userData && (
-        <DataTable
-          columns={tableColumns}
-          manualSelectColumn={selectColumn}
-          data={userData}
-          itemCount={count}
-          isSelectable
-          isPaginated
-          isFilterable
-          SelectionStatusComponent={AddLearnersSelectionStatus}
-          initialState={{
-            pageSize: 25,
-            pageIndex: 0,
-            selectedRowIds: convertToSelectedRowsObject(selectedEmails),
-          }}
-          initialTableOptions={{
-            getRowId: (row, relativeIndex, parent) => row?.uuid || (parent ? [parent.id, relativeIndex].join('.') : relativeIndex),
-          }}
-        />
-      )}
+      {errors && <Alert variant="danger">There was an error retrieving email data. Please try again later.</Alert>}
+      <DataTable
```

Comment:
Is there any visual change here? We added the table child components to appear only on load vs the original, parent table component only being shown on load.

Reply:
Yeah, because of the change in how we call the API, the parent table component has to be present for the API to be called. So I moved some things around so it's not always displaying the table.

Reply:
But no, there's no visual change.